variants/frontend-react/app/frontend/tests/HelloWorld.spec.jsx

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
This can be [dropped](https://github.com/ackama/rails-template/pull/216/files#r583919507)